packages/hooks/src/__tests__/useP2PAdvertList.spec.tsx

```diff
@@ -10,16 +10,16 @@ type TWrapper = {
 
 jest.mock('@deriv/api', () => ({
     ...jest.requireActual('@deriv/api'),
-    usePaginatedFetch: jest.fn(),
+    useQuery: jest.fn(),
 }));
 
-const mockUsePaginatedFetch = usePaginatedFetch as jest.MockedFunction<typeof usePaginatedFetch<'p2p_advert_list'>>;
+const mockUseQuery = useQuery as jest.MockedFunction<typeof useQuery<'p2p_advert_list'>>;
```

Comment:
Just a question, i thought the new `useQuery` hook was still "experimental" as farzin had described it which was being used for the next-wallet's project, is it fine to use this or useFetch for the time being 👀

Reply:
@ameerul-deriv, as seen in the screenshot below, `useFetch` is now an `alias` of `useQuery` I don't think it's experimental. 
<img width="829" alt="useFetch" src="https://github.com/binary-com/deriv-app/assets/104334373/8d7e54a1-b7da-4f72-bcbb-a63128d60c32">

Reply:
oh didnt see that, thanks for clarifying